source/compatibility.txt

```diff
@@ -33,13 +33,23 @@ The first column lists the driver version.
 
 Language Compatibility
 ----------------------
-The following compatibility table specifies the recommended versions of
+
+The following compatibility table specifies the supported versions of
 the {+driver-long+} for use with a specific version of Node.js.
 
 The first column lists the driver version.
 
 .. include:: /includes/language-compatibility-table-node.rst
 
+.. note:: The {+driver-short+} Supports Node.js Versions in the "Active LTS" Phase or Later
+
+   After release, Node.js versions have a "Current" release status for six months 
+   to give library authors time to add support for them. The {+driver-long+} 
+   officially supports Node.js versions once their release status changes 
+   from "Current" to "Active LTS". To learn more about release statues for 
```

Comment:
[Style guide on quotation marks](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#quotation-marks) says that we shouldn't use them after introducing the terms in question (current and active lts).